README.md

```diff
@@ -1,4 +1,241 @@
+#### The `#create_table` Method
+
+Above, we created a class method, `#create_table`, that crafts a SQL statement to create a songs table and give that table column names that match the attributes of an individual instance of `Song`. Why is the `#create_table` method a class method? Well, it is *not* the responsibility of an individual song to create the table it will eventually be saved into. It is the job of the class as a whole to create the table that it is mapped to. 
+
+**Top-Tip:** For strings that will take up multiple lines in your text editor, use a [heredoc](https://en.wikipedia.org/wiki/Here_document) to create a string that runs on to multiple lines. To create a heredoc, we use:
+
+`<<-` + `name of language contained in our multiline statement` + `the string, on multiple lines` + `name of language`. 
+
+You don't have to use a heredoc, it's just a helpful too for crafting long strings in Ruby. Back to our regularly scheduled programming...
+
+Now that our songs table exists, we can learn how to save data regarding individual songs into that table. 
+
+## Mapping Class Instances to Table Rows: Writing a `#save` method to `INSERT` data into the table
+
+When we say that we are saving data to our database, what data are we referring to? If individual instances of a class are "mapped" to rows in  a table, does that mean that the instances themselves, these individual Ruby objects, are saved into the database?
+
+Actually, **we are not saving Ruby objects in our database.** We are going to take the individual attributes of a given instance, in this case a song's name and album, and save *those attributes that describe an individual song* to the database, as one, single row.
+
+Let's build an instance method, `#save`, that saves a given instance of our `Song` class into the songs table of our database:
+
+```ruby
+class Song
+
+  def save
+    sql = <<- SQL
+      INSERT INTO songs (name, album) 
+      VALUES (?, ?)
+    SQL
+    
+    DB[:conn].execute(sql, self.name, self.album)
+    
+  end
+end
+``` 
+
+Let's break down the code in this method. 
+
+### The `#save` Method
+
+In order to `INSERT` data into our songs table, we need to craft a SQL `INSERT` statement. Ideally, it would look something like this:
+
+```sql
+INSERT INTO songs (name, album)
+VALUES <the given song's name>, <the given song's album>
+```
+
+Above, we used the heredoc to craft our multi-line SQL statement. How are we going to pass in, or interpolate, the name and album of a given song into our heredoc? 
+
+We use something called **bound parameters**. 
+
+#### Bound Parameters
+
+Bound parameters protect our program from getting confused by SQL injections and special characters. Instead of interpolating variables into a string of SQL, we are using the `?` characters as placeholders. Then, the special magic provided to us by the SQLite3-Ruby gem's `#execute` method will take the values we pass in as an argument and apply them as the values of the question marks. 
```

Comment:
A bit more on this. What are you protecting from? SQL Injection attacks. Link to an interesting article?